de-lib/src/main/java/org/iplantc/de/client/services/impl/AppUserServiceFacadeImpl.java

```diff
@@ -130,6 +128,9 @@ public void getApps(String id, DECallback<List<App>> callback) {
         });
     }
 
+    public void getApps(String id, DECallback<List<App>> callback) {
```

Comment:
It doesn't look like this empty method is used anymore, so it should probably be removed.